Engines/Wine/QuickScript/Steam Script/script.js

```diff
@@ -103,6 +112,8 @@ SteamScript.prototype.go = function () {
 
     wine.run(tempFile, [], null, false, true);
 
+    this.fixCertificateIssue(wine);
```

Comment:
Please add a comment here why this is required (something like "fix Wine bug 12345 (cannot download games)").